package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@eppo/js-client-sdk",
-  "version": "1.2.3",
+  "version": "1.3.0",
```

Comment:
Minor increment seems right to me since this is meant to work with a differently formatted RAC